### ISSUE TITLE
[ZOOM]: SSOe Comms Modal: Headline unreadable at 200% and 250% zoom

### ISSUE THREAD

**Issue by bridgethapner** [department-of-veterans-affairs/va.gov-team#7813]
## What happened?
Per VSP accessibility [zoom tests](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_id=21&group_order=asc) I tested the SSOe comms modal when Chrome was at 1280px width. 

- The modal was readable at 400% and 300% zoom (the top of the modal locked to the top of the screen and there was no overflow on L or R sides).
- The headline was partially cut off at 200% and fully 250% zoom and I could not scroll up to access the headline. 

### 400% zoom (success)
![Screen Shot 2020-04-09 at 11.40.15 AM.png](https://images.zenhubusercontent.com/5d8e52f61c53fd000173c5de/7c19583d-14dd-40d7-8d1b-ce66c1540971)

### 300% zoom (success)
![Screen Shot 2020-04-09 at 11.39.42 AM.png](https://images.zenhubusercontent.com/5d8e52f61c53fd000173c5de/3fc6ca40-e277-4223-a04b-3e4d4d874dcc)

### 200% zoom (headline is cut off and I cannot scroll up)
![Screen Shot 2020-04-09 at 11.41.08 AM.png](https://images.zenhubusercontent.com/5d8e52f61c53fd000173c5de/8a6b908b-70f3-4ddd-8e3c-b098aa095918)

### 250% zoom (headline is cut off and I cannot scroll up)
![Screen Shot 2020-04-09 at 11.42.01 AM.png](https://images.zenhubusercontent.com/5d8e52f61c53fd000173c5de/cbc31060-7102-4c0d-b8bb-7e172f06aee5)

## Specs:
- Device: MacBook
- Browser: Chrome, 1280px width

## Steps to Reproduce
1. Sign in on staging.va.gov with account for the first time to see modal
2. Set browser width to 1280px
3. Zoom into 200% 

## Desired behavior
Modal at 200% and 250% should lock to top of the screen as it does at 300% and 400%.

**Comment by bridgethapner** [comment]
@hinzed1127 Would love your advice here - is this something you can help with?

**Comment by 1Copenut** [comment]
Confirmed this issue using Chrome on a Windows VM.
